index.html

```diff
@@ -45,12 +32,157 @@ <h1 class="banner-text">Share, learn, code. Together.</h1>
         <p>We're a friendly bunch of coders and would love to meet new people. Share, learn and code with us at <a href="https://discord.gg/Fjur2Xa">our Discord server.</a></p>
       </div>
     </article>
-    
+
+    <!-- Members section -->
+
     <section class="members">
-      <div class="member">
-        
+      <div class="container">
+        <h1 class='members__heading'>Campfire Members</h1>
+      </div>
+      <div class="container">
+        <div class="row">
+          <div class="col-md-3 col-sm-6">
+            <div class="member">
+              <img src="members/Bakhrom.png" alt="" class="member__img">
+              <div class="member__content">
+                <h2 class="member__name">Bakhrom</h2>
+                <h3 class="member__role">Frontend Developer</h3>
+                <p class="member__desc">Hi! I'm here to go into deep dive of Frontend technologies. My skills for now:
+                  <br>
+                  # HTML5/Jade <br>
+                  # CSS3/LESS/Stylus <br>
+                  # JavaScript/jQuery/JSON/AJAX <br>
+                  # Bootstrap/Material Design <br>
+                  # Mobile First <br>
+                  # Progressive Enhancement <br>
+                  # BEM and OOCSS <br>
+                  # Agile Scrum <br>
+                </p>
+              </div>
+              <div class="social-icons">
+                <a href="http://github.com/bahromk" class="social-icons__link" target="_blank"><i class="fa fa-github"></i></a>
+                <a href="" class="social-icons__link" target="_blank"><i class="fa fa-facebook"></i></a>
```

Comment:
Please remove the placeholder social media links that don't link to one of your pages.